views/index.ejs

```diff
@@ -132,6 +135,13 @@
         <input class="inputMessage" placeholder="Type here..." />
       </div>
     </div>
+    
+    <!--   EXPORT CONTAINER    -->
```

Comment:
The idea was to make the user download the file directly.
Here it is not clear how to get the audio clip.

Reply:
I added a download link next to the audio players in my latest commits.